crates/rattler_conda_types/src/platform.rs

```diff
@@ -225,6 +225,23 @@ impl From<Platform> for &'static str {
     }
 }
 
+impl Platform {
+    /// Return the arch string for the platform
+    /// The arch is usually the part after the `-` of the platform string.
+    /// Only for 32 and 64 bit platforms the arch is `x86` and `x86_64` respectively.
+    pub fn arch(&self) -> String {
```

Comment:
Can we do this without the unwrap? especially if we get “unknown”. Maybe return option?

Reply:
Good point, especially for `noarch` it should be undefined :)

Reply:
Yeah I do think that's good, but it also makes us have the same data in two places which is what I dislike... but I can write it out and we could add a test or something.